src/vscode-bicep/src/commands/deployHelper.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { OutputChannelManager } from "../utils/OutputChannelManager";
+
+let _outputChannelManager: OutputChannelManager;
+
+export async function writeToOutputChannel(outputMessage: string) {
```

Comment:
Perhaps make these function names more specify to deployment?  The file's kind of confusing.

Reply:
Sure, updated here:
https://github.com/Azure/bicep/commit/5fd2349f36d43ef1015df3e936a3f753fd3e4bb2